src/kaleidoscope/plugin/IdleLEDsDefy.cpp

```diff
@@ -19,11 +19,7 @@
 
 #include "Communications.h"
 
-#if COMPILE_DEFY_KEYBOARD
-    #include "Defy_wireless.h"
-#elif COMPILE_RAISE2_KEYBOARD
-    #include "Raise2.h"
-#endif
+#include "kaleidoscope/device/dygma/keyboardManager/KeyboardManager.h"
```

Comment:
Why is this so long? Could we add the path to the Makefile INCDIR and use just #include "KeyboardManager.h"?

Long paths make it later difficult to re-organize the structure if needed. If it is in the Makefile, then we change the path only on one place.